doc/hooks/xfce4-dev-tools.section.md

```diff
@@ -0,0 +1,5 @@
+# `xfce.xfce4-dev-tools` {#xfce4-dev-tools}
+
+This setup hook attempts to run `xdt-autogen` in `configurePhase`. This behavior can be disabled by setting `dontUseXdtAutogenPhase` to `true`.
+
+Additionally, it appends its `share/aclocal` path to [the Autoconf macro search path](https://www.gnu.org/software/automake/manual/html_node/Macro-Search-Path.html#Modifying-the-Macro-Search-Path_003a-_002d_002daclocal_002dpath_002fACLOCAL_005fPATH).
```

Comment:
https://github.com/NixOS/nixpkgs/blob/c77dd94ccb8b3873505ef3d98dc9d3c6571f2d05/pkgs/desktops/xfce/core/xfce4-dev-tools/setup-hook.sh#L1-L5

Heh, I doubt if "envHooks" actually works [these days](https://github.com/NixOS/nixpkgs/commit/4d2b7638173afb4c0f0c12a43654bcff5c9500ab). Though in any case I think I will try removing these lines [soon](https://github.com/bobby285271/nixpkgs/blob/upd/xfce-4.20/pkgs/desktops/xfce/core/xfce4-dev-tools/setup-hook.sh) :joy:

Reply:
> Though in any case I think I will try removing these lines [soon](https://github.com/bobby285271/nixpkgs/blob/upd/xfce-4.20/pkgs/desktops/xfce/core/xfce4-dev-tools/setup-hook.sh) 😂

Should we wait until then to merge this PR?

Reply:
Removed this part of the manual section, since it doesn't work and will be removed by https://github.com/NixOS/nixpkgs/pull/351781